en/04_Changelogs/5.0.0.md

```diff
@@ -60,6 +61,35 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to SwiftMailer
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
+  - Removed the following methods from `SilverStripe\Control\Email\Email`, in most instances these have been replaced with equivalent functionality:
+    - `addAttachment()`
+    - `addAttachmentFromData()`
+    - `hasPlainPart()`
+    - `invalidateBody()`
+    - `findPlainPart()`
+    - `generatePlainPartFromBody()`
+    - `getBCC()`
+    - `getCC()`
+    - `getFailedRecipients()`
+    - `getFrom()`
+    - `getReturnPath()`
+    - `getSender()`
+    - `getSwiftMessage()`
+    - `getTo()`
+    - `setSwiftMessage()`
+    - `setFailedRecipients()`
+    - `BaseURL()`
+    - `IsEmail()`
+  - Changed return types on the following methods in `SilverStripe\Control\Email\Email`
+    - `send()`
+    - `sendPlain()`
+  - Changed visibility on the following methods to `private` in `SilverStripe\Control\Email\Email`
+    - `mergeConfiguredEmails()`
```

Comment:
This section is only for _removed_ API.
Mention these below, in the email-specific section.

Reply:
Updated